js/views/endScreen.js

```diff
@@ -27,11 +46,40 @@ var EndScreen = React.createClass({
       opacity: this.props.skinConfig.endScreen.replayIconStyle.opacity
     };
 
+    var titleStyle = {
+      color: this.props.skinConfig.startScreen.titleFont.color
```

Comment:
We are reusing the color in pause screen, so I thought it makes sense if all three screens have the same color.